lib/miq_automation_engine/service_models/miq_ae_service_vm_or_template.rb

```diff
@@ -41,6 +41,7 @@ class MiqAeServiceVmOrTemplate < MiqAeServiceModelBase
     expose :compliances,           :association => true
     expose :last_compliance,       :association => true
     expose :ems_events,            :association => true
+    expose :evacuate,              :override_return => nil
```

Comment:
@andyvesel I think the idea is to not use `:override_return` and allow the data to pass back to the caller.  Any idea what the return value would be here?